infra/ansible/roles/build_srcs/tasks/main.yaml

```diff
@@ -122,10 +123,14 @@
         #   torch-2.6.0.dev-cp310-cp310-linux_x86_64.whl
         # becomes
         #   torch-2.6.0.dev20241206-cp310-cp310-linux_x86_64.whl
-        newf="${f/.dev-/.dev${current_date}-}"
-
-        # Copy the file to the new filename.
-        cp "$f" "$newf"
+        datedf="${f/.dev-/.dev${current_date}-}"
+        # Remove the version number. For example:
+        #   torch.dev-cp310-cp310-linux_x86_64.whl
+        unversionedf="${f/-(.*?)\.dev/.dev/1}"
```

Comment:
Can we just use `unversionedf="${f/-[0-9].[0-9].[0-9].dev/.dev}"`?

The existing line of the script didn't work for me.

Reply:
Updated to use sed. Works for me now in local testing